### ISSUE TITLE
Test Accessibility: Products Page

### ISSUE THREAD

**Issue by naragurgel** [naragurgel/serenity_oils_aroma#50]
# Steps to take

1. Open a new incognito tab in the web browser.
2. Enter the URL of the [products](https://serenity-oils-3beb26e98281.herokuapp.com/products/) page. 
3. Right-click anywhere on the page and select "Inspect" to open the browser's developer tools.
4. In the developer tools panel, navigate to the "Audits" tab.
5. Click on the "Run audits" button to initiate the Lighthouse audit.

# Expected Outcome

- [x]  Accessibility should be 90+

**Comment by naragurgel** [comment]
![image](https://github.com/naragurgel/serenity_oils_aroma/assets/112726044/7e853de7-2e7f-4436-86c8-1e93802cebfc)
